src/PointwiseFunctions/AnalyticSolutions/EinsteinSolutions/TOV.hpp

```diff
@@ -0,0 +1,67 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <memory>
+
+#include "DataStructures/DataVector.hpp"
+#include "NumericalAlgorithms/Interpolation/BarycentricRational.hpp"
+#include "PointwiseFunctions/EquationsOfState/EquationOfState.hpp"  // IWYU pragma: keep
+
+namespace PUP {
+class er;
+} // namespace PUP
+
+// IWYU pragma: no_forward_declare EquationsOfState::EquationOfState
+
+/// Contains functions for TOV solver
+namespace tov {
```

Comment:
Given the directory this is currently in the namespace should be `EinsteinSolutions`